benchmarks/templates/benchmarks/tutorials/models/deepdive_1.html

```diff
@@ -130,155 +112,148 @@ <h3 class="benefits_heading is-size-3-mobile">Part 4: Exploring the  __init__.py
+        with the actual model identifier you used to add your mode to the Brain-Score model registry in __init__.py.)
+
+    <pre class="modified_1"><code >
+python brainscore_vision score --model_identifier='YOUR_MODEL_IDENTIFIER' --benchmark_identifier='MajajHong2015public.IT-pls'
     </code></pre>
 
+    <p class="benefits_info is-size-5-mobile shorter">
+        Upon scoring completion, you should get a message like below:
+    </p>
+    <pre class="modified_1"><code >
+&lt;xarray.Score ()&gt;
+array(0.07637264)
+Attributes:
+    error:                 &lt;xarray.Score ()&gt;\narray(0.00548197)
+    raw:                   &lt;xarray.Score ()&gt;\narray(0.22545106)\nAttributes:\...
+    ceiling:               &lt;xarray.DataArray ()&gt;\narray(0.81579938)\nAttribut...
+    model_identifier:      pixels
+    benchmark_identifier:  MajajHong2015public.IT-pls
+    comment:               layers: {'IT': 'pixels'}
 
 
-    <div class="columns is-tablet is-variable is-1-tablet">
-        <div class="column is-one-half tutorial_text">
-           <p class="benefits_info is-size-5-mobile shorter top_adjust">
-                Once you receive this message, you could rezip (after you save everything, of course) your package, and you
-                would be good to submit your model.
-           </p>
-            <p class="benefits_info is-size-5-mobile shorter ">
-                When you submit an actual (non-tutorial) model, you will receive an email with your results
-                within 24 hours (most of the time it only takes 2-3 hours to score).
-                If you would like to explore a custom model's submission package, please visit the next
-                Deep Dive <a href="http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}/tutorials/models/deepdive_2">here</a>.
-            </p>
-        </div>
+Process finished with exit code 0
+    </code></pre>
+    <p class="benefits_info is-size-5-mobile shorter">
+        These numbers are broken down in the <a href="https://www.brain-score.org/tutorials/models/quickstart">Quickstart Tutorial</a>,
+        but the main point here is that if you get a message that looks like the above (likely with different numbers) then
+        your model is ready to submit.  (Please note that compute times may vary significantly depending on your local hardware setup.)
+        Once you successfully run your model locally, you can rezip your package, and you would be ready to submit your model.
+    </p>
+    <p class="benefits_info is-size-5-mobile shorter ">
+        When you submit an actual (non-tutorial) model, you'll receive an email with your results
+        within 24 hours. If you would like to explore a custom model submission package, please visit <a href="http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}/tutorials/models/deepdive_2">Deep Dive #2</a>.
+    </p>
```

Comment:
```suggestion
<div class="columns is-tablet is-variable is-1-tablet">
        <div class="column is-one-half tutorial_text">
            <p class="benefits_info is-size-5-mobile shorter">
                These numbers are broken down in the <a href="https://www.brain-score.org/tutorials/models/quickstart">Quickstart Tutorial</a>,
                but the main point here is that if you get a message that looks like the above (likely with different numbers) then
                your model is ready to submit.  (Please note that compute times may vary significantly depending on your local hardware setup.)
                Once you successfully run your model locally, you can rezip your package, and you would be ready to submit your model.
            </p>
            <p class="benefits_info is-size-5-mobile shorter ">
                When you submit an actual (non-tutorial) model, you'll receive an email with your results
                within 24 hours. If you would like to explore a custom model submission package, please visit <a href="http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}/tutorials/models/deepdive_2">Deep Dive #2</a>.
            </p>
    </div>
```

**IMPORTANT** Added two `<div>` at 247 and 248 and a `</div>` at 259. Need these for the correct formatting of the balloon image and the correct placement of the right sidebar.